src/index.html

```diff
@@ -16,17 +16,25 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="body">
+    <div class="input input__icon input__icon--big">
+      <input
+        class="input__big"
+        type="text"
+        data-qa="keypress big"
+        placeholder="Try “Los Angeles“"
+      />
+      <!-- <span class="input__icon"></span> -->
+    </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="input input__icon input__icon--small">
+      <input
+        type="text"
+        data-qa="hover small"
+        placeholder="Try “Los Angeles“"
+        class="input__small"
+      />
+      <!-- <span class="input__icon"></span> -->
+    </div>
```

Comment:
Similar to the previous comment, the class names `input__icon` and `input__icon--small` may not accurately represent the content if there is no icon present.